developer_manual/app/changelog.rst

```diff
@@ -24,6 +39,34 @@ This is a deprecation roadmap which lists all current deprecation targets and wi
 
 .. note:: Deprecations on interfaces also affect the implementing classes!
 
+11.1
```

Comment:
Do we? We have a 3 year deprecation period which means 8.1 + 3 = 11.1

At least per http://karlitschek.de/2015/06/building-a-platform/:

> I see three years as the sweet spot before we can remove a deprecated API, so developers and users have enough time to migrate. Of course, security can be an exception after careful considerations. It can also be helpful to create tools that help developers migrate to a new API.